test/system/deleted-files.spec.ts

```diff
@@ -65,3 +65,78 @@ describe("deleted files", () => {
     ).toEqual(1);
   });
 });
+
+describe("Deleted Folders", () => {
+  const getFixture = (fixturePath) =>
+    path.join(__dirname, "../fixtures/docker-archives", fixturePath);
+
+  /*
+    The image in the example below was built with the following Dockerfile:
```

Comment:
this is unrelated to your change, but I think it's important:

could we submit the Dockerfiles to the test fixtures and maybe even generate the image tars on-demand? It would make these tests a lot more maintainable. Right now re rely on comments to understand how an image tar was generated

Reply:
I do think it's a little out of scope.

Regarding generating the test fixtures on demand, that's not a bad idea. It would require adding docker as a dependency though. Do you think that's worth it?

Reply:
i agree that this makes sense, and is partially already done for example for the [Go binary tests](https://github.com/snyk/snyk-docker-plugin/blob/fcb30861bbc2075bf364abaa719ea1163cd7bdff/test/fixtures/go-binaries/source/build.sh#L1).

(The test itself runs nightly and generates fresh Go binaries based on `go:latest` to ensure new releases don't break: https://github.com/snyk/snyk-docker-plugin/blob/fcb30861bbc2075bf364abaa719ea1163cd7bdff/.circleci/config.yml#L209)

Reply:
this would obviously not be part of this PR, but I think it's still worth doing... maybe by the next team